src/batch/src/rpc/service/task_service.rs

```diff
@@ -158,12 +160,12 @@ impl TaskService for BatchServiceImpl {
             // therefore we would only have one data output.
             output_id: 0,
         };
-        let mut output = task.get_task_output(&pb_task_output_id).map_err(|e| {
+        let mut output = task.get_task_output(&pb_task_output_id).inspect_err(|e| {
             error!(
-                "failed to get task output of Task {:?} in local execution mode",
-                task_id
+                error = %e.as_report(),
+                ?task_id,
+                "failed to get task output in local execution mode",
             );
-            e
         })?;
```

Comment:
Prefer `inspect_err` if not going to change the error variant. For example, logging it out.